network/firewall.go

```diff
@@ -46,13 +46,13 @@ func (c *FirewallClient) ListRules(ctx context.Context, _ *ListRulesInput) ([]*F
 		defer respReader.Close()
 	}
 	if err != nil {
-		return nil, errwrap.Wrapf("Error executing ListRules request: {{err}}", err)
+		return nil, errors.Wrap(err, "unable to list rules")
```

Comment:
In all of these error messages can we change it to `unable to list firewall rules`, basically:

`s/rules/firewall rules/g`
  
I'm positive we're going to add additional types of rules and it would be good to keep error messages unique.